### ISSUE TITLE
Python package name conflict

### ISSUE THREAD

**Issue by JLVarjo** [openbabel/openbabel#1843]
When installing OpenBabel python bindings via pip, the name to be used is "openbabel". However, the package ends up to pip listing as "openbabel-python". This causes problems in freezing the package list with pip, since the package under that name will not be found when installing the list.

**Comment by baoilleach** [comment]
I don't understand the problem. Could you describe in more detail what you are trying to do and why it isn't working?

**Comment by JLVarjo** [comment]
Strange. Could this be platform-dependent somehow? My setup is Windows 10 / Python 3.6.5 (64-bit) / pip 10.0.1, and I have seen this on two other Windows machines as well. Installing uses the following wheel:

```
C:\>pip install openbabel
Collecting openbabel
  Using cached https://files.pythonhosted.org/packages/b1/be/7d35e887a7b6c79aa6e658cddcf32430cea202bfe7301deff7addfee05e1/openbabel-2.4.1-cp36-cp36m-win_amd64.whl
Installing collected packages: openbabel
Successfully installed openbabel
```
 
And subsequently pip freeze says

`openbabel-python==2.4.1`

**Comment by baoilleach** [comment]
I created the Windows versions using different files, and so I've probably made a mistake. I'll assign this to me...

**Comment by baoilleach** [comment]
Bump.

**Comment by baoilleach** [comment]
I know it's a bit late, but I'm in the process of creating a release and will make this change. I'll leave open until I've actually done it.

**Comment by mhangaard** [comment]
I did `snap install openbabel` and that works just fine

**Comment by baoilleach** [comment]
Hi @mhangaard, you should file a separate bug or email our mailing list. The issue you describe is unrelated to this Windows Python package problem.

**Comment by baoilleach** [comment]
I fixed this before the 3.0 release.
